widget/embedded/src/pages/WalletsPage.tsx

```diff
@@ -64,17 +76,41 @@ export function WalletsPage() {
   const selectedWalletState =
     selectedWallet?.state || WalletState.NOT_INSTALLED;
 
+  const filteredBlockchains = filterBlockchainsByWalletTypes(list, blockchains);
```

Comment:
Until the metadata response is received, there will be no wallet list, and consequently, no skeleton for the wallet list either :
![3](https://github.com/rango-exchange/rango-client/assets/120931880/5f8905cd-b0ec-4198-b722-249090d86dab)

Reply:
This wasn't an issue for me. I appreciate you checking once more.
![Screenshot from 2024-05-14 11-24-06](https://github.com/rango-exchange/rango-client/assets/120931936/aa4f993f-dfeb-4797-b44b-9b087a458714)